src/uproxy.ts

```diff
@@ -104,6 +103,33 @@ module uProxy {
     data :Object;
   }
 
+  // The different states that uProxy consent can be in w.r.t. a peer. These
+  // are the values that get sent or received on the wire.
+  export interface WireState {
```

Comment:
Can we call this ConsentWireState?  Now that it is in the uProxy module it's not as clear what just "WireState" means